tooltester/src/main/resources/JenkinsReadme.md

```diff
@@ -1,21 +1,37 @@
-Master Setup:
+
+# Master Setup:
 1.  Install the Jenkins pipeline suite
 2.  Create a pipeline called PipelineTest
 3.  Copy the contents of resources/PipelineTest.groovy into the Pipeline Script textbox
 4.  Check the checkbox:  "This project is parameterized"
 5.  Create 5 String Parameters with the names: "URL", "Tag", "DescriptorPath", "ParameterPath", "DockerfilePath"
-6.  sudo apt-get install ansible
+6.  Install ansible
+    sudo apt-get install software-properties-common
+    sudo apt-add-repository ppa:ansible/ansible
+    sudo apt-get update
+    sudo apt-get install ansible
```

Comment:
Document a version of ansible this was tested with (ansible seems to release pretty often)